src/SearchIndexer/Sfa.Das.Sas.Infrastructure/Elasticsearch/ElasticsearchMapper.cs

```diff
@@ -57,12 +57,13 @@ public FrameworkDocument CreateFrameworkDocument(FrameworkMetaData frameworkMeta
 
             var doc = new FrameworkDocument
             {
-                FrameworkId = $"{frameworkMetaData.FworkCode}{MapToLevelFromProgType(frameworkMetaData.ProgType)}{frameworkMetaData.PwayCode}",
```

Comment:
We are still using level as id when creating FrameworkProvider (Line: 137 about..)
They need to be the same right?
@mapaux